.github/workflows/desktop.yml

```diff
@@ -74,9 +74,13 @@ jobs:
             architecture: "x86"
           # Xcode excludes -- allow only one on osx and linux
           - os: ubuntu-latest
```

Comment:
why there's extra two entries?

Reply:
This makes it so it doesn't repeat the linux/windows jobs even if the matrix contains xcode 11.7, 12.4, and 12.5.1. (previously it was omitting 12.4, which is the actual version we use now, so this swaps it to the others - previously there was a bug where 11.7 was missing here.)

Reply:
To clarify: the expanded matrix will have the following (simplified) list of targets:

- macos - xcode 11.7
- windows - xcode 11.7
- ubuntu - xcode 11.7
- macos - xcode 12.4
- windows - xcode 12.4
- ubuntu - xcode 12.4
- macos - xcode 12.5.1
- windows - xcode 12.5.1
- ubuntu - xcode 12.5.1
These "excludes" entries ensure that only one set of ubuntu and windows will be run. And it can't exclude 12.4, because 12.4 is now the one that runs in the non-expanded matrix.